### PR TITLE
[IT-1101] Add templates to setup AWS master member guardduty

### PR DESCRIPTION
**Pull request by zaro0508**
These templates will allow org-formation to deploy guardduty in a
master member configuration[1].  All templates are just copied
from the ofn reference project[2]

[1] https://docs.aws.amazon.com/guardduty/latest/ug/guardduty_accounts.html
[2] https://github.com/org-formation/org-formation-reference/tree/master/src/templates/070-guard-duty

### CODE REVIEW

File: templates/GuardDuty/trusted-ips-bucket.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Sid: GuardDutyRead
+            Effect: Allow
+            Principal:
+              Service: guardduty.amazonaws.com
+            Action:
+              - s3:GetObject
+            Resource: !Sub '${TrustedIpsBucket.Arn}/trusted_ips.txt'
```
Comment (ahayden) [inline templates/GuardDuty/trusted-ips-bucket.yaml:47]: Maybe it doesn't matter if the trusted IPs list is empty... but do want to have a list like this?

Comment (zaro0508) [reply templates/GuardDuty/trusted-ips-bucket.yaml:47]: do you have a concern with it being here?  if not then lets just add as an optional feature.  if you have a concern later you are free to remove it if you want.

**Review by ahayden — APPROVED** [review]
All looks good. Where would we subscribe to the notification topics? IE to get an email

**Comment by zaro0508** [comment]
> All looks good. Where would we subscribe to the notification topics? IE to get an email

it's here @ahayden: 
https://github.com/Sage-Bionetworks/aws-infra/pull/288/files#diff-afc5705ba1fe086bdf9b9ab4c0f51b261dc6ed4e07c0119eeb54c75d5609e9b8R29

**Pull request merged**